mods/cnc/maps/nod08a/rules.yaml

```diff
@@ -138,26 +138,6 @@ A10.IN:
 	Armament@BOMBS:
 		Weapon: Napalm.in
 
-airstrike.proxy:
-	AlwaysVisible:
-	AirstrikePower:
-		Icon: airstrike
-		StartFullyCharged: True
-		ChargeTime: 120
-		SquadSize: 3
-		QuantizedFacings: 8
-		Description: Air Strike
-		LongDesc: Deploy an aerial napalm strike.\nBurns buildings and infantry along a line.
-		EndChargeSound: airredy1.aud
-		SelectTargetSound: select1.aud
-		InsufficientPowerSound: nopower1.aud
-		IncomingSound: enemya.aud
-		UnitType: a10.in
```

Comment:
It is indeed, thanks. I already wondered why my buildings aren't destroyed by the airstrike, but then thought it's because I used the debug menu. Well, I guess the first mistake was at "I thought".